client/profile-wizard/index.js

```diff
@@ -134,8 +140,10 @@ class ProfileWizard extends Component {
 				profileItems.product_types !== null,
 		} );
 		steps.push( {
-			key: 'business-details',
-			container: BusinessDetails,
+			key: selectiveBundleInstallSegmentation
```

Comment:
The reason we have to check the segmentation this high up, is because there is analytics tracking that happens as you progress through the wizard steps, we have to change the step name conditionally here to make sure the tracked event is given the right name. This is a result of the design of the existing code, I don't think theres a better way to do this without a significant refactor.